File: 0724-find-pivot-index/0724-find-pivot-index.cpp
class Solution {
public:
    int leftSum(vector<int>& arr, int target) {
        int sum = 0;
        for (int i = 0; i < target; i++) {
                sum += arr[i];
            
        }
        return sum;
    }

    int rightSum(vector<int>& arr, int target) {
        int sum = 0;
        for (int i = target+1; i <arr.size(); i++) {
            
                sum += arr[i];
            
        }
        return sum;
    }
    int pivotIndex(vector<int>& nums) {
        for (int i = 0; i < nums.size(); i++) {
            if (leftSum(nums,i) == rightSum(nums,i)) {
                return i;
            }
        }
        return -1;
    }
};

File: 0724-find-pivot-index/0724-find-pivot-index.java
class Solution {

    int leftSum(int[] nums , int j){
        int sum=0;
        for(int i=0;i<j;i++){
            sum += nums[i];
        }
     return sum;
    }
     int rightSum(int[] nums , int j){
        int sum=0;
        for(int i=j+1;i<nums.length;i++){
            sum += nums[i];
        }
     return sum;
    }
    public int pivotIndex(int[] nums) {
        for(int i=0;i<nums.length;i++){
            if(leftSum(nums,i)==rightSum(nums,i)) return i;
        }
    return -1;  
    }
}